[DX-1343] Replace master/slave content where possible, for tier 1 compliance with inclusive naming (#4641)
TykTechnologies/tyk-docs: tyk-docs/content/planning-for-production/ensure-high-availability/uptime-tests.md

### How do the uptime tests work?

> When uptime tests are added into a Tyk cluster, a single node will elect itself as primary. Primary nodes stay in this state using a dead man's switch, by keeping a key active in Redis. Primaries are re-elected or confirmed every few seconds. If one node stops or fails, another can detect the failure and elect itself as the primary node.

Reviewer: ```suggestion
When uptime tests are added into a Tyk cluster, a single Gateway will elect itself as primary. Gateways remain as primary using a dead man's switch, by keeping a key active in Redis. Primaries are re-elected or confirmed every few seconds. If one Gateway stops or fails, another can detect the failure and elect itself as the primary.
```

Try to move away from the term "node" as it is not defined anywhere - use Gateway.

> The primary node will then run the uptime tests allocated to the cluster (shard group).

Reviewer: ```suggestion
The primary Gateway will then run the uptime tests allocated to the cluster (shard group).
```

> The node running the uptime test will have a worker pool defined so that it can execute tests simultaneously every few seconds determined by a node-configurable interval loop. Depending on how many uptime tests are being run, this worker pool should be increased or decreased as needed.

Reviewer: ```suggestion
The Gateway running the uptime test will have a worker pool defined so that it can execute tests simultaneously every few seconds determined by a Gateway-configurable interval loop. Depending on how many uptime tests are being run, this worker pool should be increased or decreased as needed.
```

```

> *   `disable`: When set to `false` this tells Tyk to run uptime tests, if you do not want any uptime tests to run on a node, set it to `true` and they will be disabled on those nodes (this could be useful if you are running uptime tests in a separate group of Tyk instances).

Reviewer: ```suggestion
*   `disable`: When set to `false` this tells Tyk to run uptime tests, if you do not want any uptime tests to run on a Gateway, set it to `true` and they will be disabled on those Gateways (this could be useful if you are running uptime tests in a separate group of Tyk instances).
```

> *   `poller_group`: This field is used to define a different group of uptime tests. All the gateways that have the same `poller_group`, will be candidate to be elected as the primary node of its group. This could be useful if you are running uptime tests in a segmented or sharded group of Tyk instances.

Reviewer: ```suggestion
*   `poller_group`: This field is used to define a different group of uptime tests. All the gateways that have the same `poller_group`, will be candidate to be elected as the primary Gateway of its group. This could be useful if you are running uptime tests in a segmented or sharded group of Tyk instances.
```
